src/rendition-mixin.js

```diff
@@ -31,7 +31,7 @@ const enableFunction = (loader, playlistUri, changePlaylistFn, enable) => {
     // Ensure the outside world knows about our changes
     changePlaylistFn();
   }
-
+  loader.trigger('hlsrenditiondisabled');
```

Comment:
You are right about that. But there is one more problem with this trigger, it seems I can't trigger event on the `loader` passed in. It says `loader.trigger is not a function`, isn't it supposed to be used like a regular playlist loader?